### PR TITLE
Issue23 read wav

### PR DESCRIPTION
**Pull request by jhmigueles**
Small changes to the readWav function to fix the detection of the header and the reading of the wav files.

### CODE REVIEW

File: R/readWav.R [hunk 1]
```diff
@@ -8,26 +8,36 @@ readWav = function(filename, start = 1, end = 100, units = "minutes") {
...
+      } else {
+        header = header_rownames
+      }
     }
     Nlines = Nlines - 1
-    if (Nlines == 10) print("Error: wav file header not recognized in function g.wavread")
+    if (Nlines == 10) print("Error: wav file header not recognized in function readWav from the GGIRread package")
   }
-  scale3position = grep("Scale-3", header)
+  if (length(grep("Scale-3",header)) > 0) scale3position = grep("Scale-3", header)
```
Comment (vincentvanhees) [inline R/readWav.R:40]: Did you observe cases where "Scale-3" was not present in the header?
Note that the while statement above already checks for "Scale-3" to be present. So, I think this if-statement is not needed.

File: R/readWav.R [hunk 2]
```diff
@@ -8,26 +8,36 @@ readWav = function(filename, start = 1, end = 100, units = "minutes") {
   B = as.data.frame(B, stringsAsFactors = TRUE)
   #-------------------------------------------------------
   # extract info from header: fileEncoding does not seem to be consistent, so try variants
-  header = c()
-  Nlines = 16
-  while (length(header) == 0 | length(grep("Scale-3",header)) == 0 |
+  header = header_rownames = c()
+  Nlines = 18
+  options(warn = -1)
```
Comment (vincentvanhees) [inline R/readWav.R:13]: An alternative approach to `options(warn  = -1)` could be to use `suppressWarnings()` around each of the 4 `read.csv()` calls. In that way it is more clear which warnings we are trying to suppress.

**Review by vincentvanhees — COMMENTED** [review]
Thanks Jairo, looks good. Two minor suggestions.
Feel free to add yourself to the DESCRIPTION file as contributor. I assumed you had write access for this repository, I will now add it.

File: R/readWav.R (hunk 1 above)
Comment (jhmigueles) [reply R/readWav.R:40]: No, not really, I was not familiar with the wav heading and in my first attempts I did not realize that the scale-3 was always available.

File: R/readWav.R (hunk 2 above)
Comment (jhmigueles) [reply R/readWav.R]: Done =)

**Comment by jhmigueles** [comment]
I think this is now ready for merging, just finished applying the changes you proposed.

**Pull request merged**